spec/requests/show.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Show', type: :request do
+  describe 'GET /companies/:id' do
```

Comment:
This would be enough:
```suggestion
RSpec.describe 'GET /companies/:id', type: :request do
```